Library/Homebrew/cli_parser.rb

```diff
@@ -60,17 +60,28 @@ def conflicts(*options)
         @conflicts << options.map { |option| option_to_name(option) }
       end
 
-      def option_to_name(name)
-        name.sub(/\A--?/, "").tr("-", "_").delete("=")
+      def option_to_name(option)
+        option.sub(/\A--?/, "")
+              .tr("-", "_")
+              .delete("=")
+      end
+
+      def name_to_option(name)
+        if name.length == 1
+          "-#{name}"
+        else
+          "--#{name}"
+        end
       end
 
       def option_to_description(*names)
         names.map { |name| name.to_s.sub(/\A--?/, "").tr("-", " ") }.max
```

Comment:
I know this code isn't directly part of this refactor, but may it make sense to make it more closely match the above syntax? Any thoughts @MikeMcQuaid?

Reply:
@GauthamGoli, do we even need this? We are using `OptionParser` only to parse arguments and not to display descriptions.

Reply:
@GauthamGoli Given we're not using descriptions anywhere yet I'd be 👍 in deleting the description code until it's used (or using it in some places now)

Reply:
👍 For removal if it's not being utilized.

Reply:
It seems from https://github.com/Homebrew/brew/pull/4271#issuecomment-394178063 it will be used very shortly so let's leave it as-is for now.

Reply:
Sorry. I missed this notification. Yeah. I'll be working on this shortly, don't remove it.